### PR TITLE
DR-1470 subproject restructuring/imports

### PR DESCRIPTION
**Pull request by myessail**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -290,23 +286,12 @@ task unpack(type: Copy) {
...
 
-def excludePattern = 'generated-src/antlr/main/bio/terra/grammar/'
-def excludeGrammarPattern = '${buildDir}/' + excludePattern + '*'
-
-
-tasks.withType(Checkstyle) {
-    exclude excludeGrammarPattern
-    exclude "${buildDir}/generated-src/antlr/main/bio/terra/grammar/*"
-}
-
```
Comment (fboulnois) [inline build.gradle:309]: Does this mess with the antlr functionality?

**Review by fboulnois — CHANGES_REQUESTED** [review]
A few questions:

**Comment by fboulnois** [comment]
It lost my other questions. I'll try again. :(

File: build.gradle [hunk 2]
```diff
@@ -160,16 +154,14 @@ configurations {
 
 dependencies {
     // TODO: Unpack the "starter" and include just what we use
-
+    compile 'ch.qos.logback:logback-classic:1.2.3'       // Logger
```
Comment (fboulnois) [inline build.gradle]: I think @ddietterich removed this in a previous commit. Do we still need it?

**Review by fboulnois — CHANGES_REQUESTED** [review]
Additional question:

File: build.gradle (hunk 1 above)
Comment (myessail) [reply build.gradle:309]: nope! this is accounted for already in `checkstyle-suppressions.xml`

**Review by fboulnois — APPROVED** [review]
lgtm assuming tests pass.

**Review by RoriCremer — APPROVED** [review]

**Pull request merged**